src/Sentry.Maui/SentryMauiOptions.cs

```diff
@@ -23,18 +23,14 @@ public SentryMauiOptions()
 #if !PLATFORM_NEUTRAL
         CacheDirectoryPath = Microsoft.Maui.Storage.FileSystem.CacheDirectory;
 #endif
-        AddDefaultEventBinder<MauiButtonEventsBinder>();
-        AddDefaultEventBinder<MauiImageButtonEventsBinder>();
-        AddDefaultEventBinder<MauiGestureRecognizerEventsBinder>();
-        AddDefaultEventBinder<MauiVisualElementEventsBinder>();
     }
 
-    internal List<IMauiElementEventBinderRegistration> DefaultEventBinders { get; } = [];
+    internal List<IMauiElementEventBinderRegistration> InjectedEventBinders { get; } = [];
```

Comment:
Potentially we could call this property `EventBindersFromIntegrations`... at the moment, there's only one.